test/JDBC/expected/13_4__preparation__bbf_view_def_upgrade-vu-prepare.out

```diff
@@ -0,0 +1,13 @@
+CREATE VIEW view_3135_upd AS SELECT 1;
+GO
+
+CREATE VIEW dep_view_3135 AS SELECT * FROM sys.babelfish_view_def;
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: relation "sys.babelfish_view_def" does not exist)~~
+
+
+CREATE VIEW dep_view_3135_2 AS SELECT * FROM information_schema.views;
```

Comment:
Can we please add comments for this and what is expected (If fail, then why?)

Reply:
Added.

Reply:
Added.